_integration-schemas/codat/v1/events.md

```diff
@@ -0,0 +1,38 @@
+---
+tap: "codat"
+version: "1"
+key: "event"
+
+name: "events"
+doc-link: "https://docs.codat.io/reference/reports#reports_companyevents"
+singer-schema: "https://github.com/singer-io/tap-codat/blob/master/tap_codat/schemas/events.json"
+description: |
+The {{ table.name }} table contains information about a given company's events in your {{ integration.display_name }} account.
```

Comment:
This is going to prevent the site from building. You should've received an error in the Terminal for this.